src/components/validators/project-name-available-validator.js

```diff
@@ -6,6 +6,9 @@
       return {
         restrict: 'A',
         require: 'ngModel',
+        scope: {
+          organizationId: "="
```

Comment:
Come to think of which is more prefered in the project, in such case?

As-it or `@` here with `organization-id="{{vm.currentOrganization.id}}"` in the `src/app/project/add.tpl.html` file?